test/verify/check-storage-vdo

```diff
@@ -27,6 +27,7 @@ from testlib import *
 SUPPORTED_OS = ["centos-8-stream", "rhel-8-4", "rhel-8-4-distropkg", "rhel-8-5", "rhel-8-5-distropkg"]
 
 
+@skipImage("no vdo package yet on RHEL 9, https://bugzilla.redhat.com/show_bug.cgi?id=1916216", "rhel-9-0", "rhel-9-0-distropkg")
```

Comment:
Hmm, This would be better as naughty I guess as we we almost certain to forget this here and not test even when vdo will be available.

Reply:
Ah right, just read the BZ. This is fine and when we get reply that it will be build then we should move it to naughty.